assets/components/GeoMap.js

```diff
@@ -1,6 +1,22 @@
 export const GeoMap = () => {
   return `
     <div id="map"></div>
+    <div id="map-controls">
+      <div class="map-controls-zoom">
+        <button class="map-controls-zoom-btn" id="mapZoomIn">
+          <img src="/assets/icons/plus.svg" alt="+" />
```

Comment:
<i class="icon-plus"></i>

use i tag instead of img, to change the icon color and double-check the styles. There are some inconsistencies in the icon size but please make sure to match all the styles.